backend/src/main/java/com/htecgroup/skynest/util/JwtUtils.java

```diff
@@ -132,4 +138,77 @@ public String buildEmail(String email, String link) {
         + "\n"
         + "</div></div>";
   }
+
+  public String buildPasswordResetEmail(String email, String link) {
+    return "<div style=\"font-family:Helvetica,Arial,sans-serif;font-size:16px;margin:0;color:#0b0c0c\">\n"
```

Comment:
Is there a way to move this huge String in some file, and to read it from file ?

Reply:
There should be, we plan to do the same with the verification email.

Reply:
Then create a ticket for that, so that you dont forget it :)

Reply:
I made sure to add it as part of the refactor task checklist